From e5af768419680639d585d1b119639b984ca4633c Mon Sep 17 00:00:00 2001
From: Fotios Sioutis <sfotis@gmail.com>
Date: Fri, 16 Dec 2011 12:15:21 +0200
Subject: [PATCH] atol of cstdlib must be changed to std::atol (bcc)

since upstream changed "stdlib.h" to "cstdlib" now all functions
declared inside standard library become members of std namespace.
---
 src/OSD/OSD_MAllocHook.cxx |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/OSD/OSD_MAllocHook.cxx b/src/OSD/OSD_MAllocHook.cxx
index ef41e0d..ed49e10 100644
--- a/src/OSD/OSD_MAllocHook.cxx
+++ b/src/OSD/OSD_MAllocHook.cxx
@@ -291,10 +291,10 @@ Standard_Boolean OSD_MAllocHook::LogFileHandler::MakeReport
     while (*pStr != ' ' && *pStr) pStr++;
     *pStr++ = '\0';
     while (*pStr == ' ' && *pStr) pStr++;
-    aReqNum = atol(pStr);
+    aReqNum = std::atol(pStr);
     while (*pStr != ' ' && *pStr) pStr++;
     while (*pStr == ' ' && *pStr) pStr++;
-    aSize = atol(pStr);
+    aSize = std::atol(pStr);
     Standard_Boolean isAlloc = Standard_False;
     if (strcmp(aType, "alloc") == 0)
     {
-- 
1.7.9.1

